src/python/setup.py.in

```diff
@@ -24,18 +25,25 @@ package_name = '@TARGET_NAME@'
 
 def _onnxruntime_dependency() -> str:
     dependency = None
+    ort_version = os.environ.get("ONNXRUNTIME_VERSION", "")
+    is_nightly = True if "dev" in ort_version else False
+
     if package_name == "onnxruntime-genai":
-        dependency = "onnxruntime"
+        dependency = "onnxruntime" if not is_nightly else "ort-nightly"
+
+        import platform
+        if platform.machine() == "ARM64":
```

Comment:
Only Qualcomm device have QNN capability. Should we assume this?

Reply:
ARM64 Vendors like MediaTek and HiSilicon do not have QNN in their SoCs.

Reply:
The reason for doing this is because the onnxruntime python package does not have windows arm64 whls. The windows arm64 wheels are only available in onnxruntime-qnn.
I think I am missing a check for windows os here.